setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='ocrd_tesserocr',
-    version='0.4.0',
+    version='0.4.1',
```

Comment:
Better to do that in explicit version bumping commits.

Reply:
Understood. @kba, should I revert this in the PR? Do you want to do the merge and the versioning commit along with the new release?